src/i18n/de-DE.json

```diff
@@ -1100,10 +1100,10 @@
   "setEmail.headline": "E-Mail-Adresse festlegen",
   "setEmail.tryAgain": "Erneut versuchen",
   "setEntropy.continue": "Weiter",
-  "setEntropy.headline": "Entropie erzeugen",
-  "setEntropy.moreEntropyNeeded": "Bitte bewegen Sie Ihre Maus weiterhin innerhalb des Fensters, um genügend Entropie zu erzeugen.",
-  "setEntropy.subheadline": "Bewegen Sie hierfür Ihre Maus möglichst zufällig in dem weißen Fenster, bis der Fortschrittsbalken zu 100 % gefüllt ist. Auf diese Weise tragen Sie dazu bei, die Qualität der Zufallszahlen, die für die Erzeugung Ihrer kryptographischen Langzeit-Geheimnisse verwendet werden, zu verbessern und damit die Sicherheit dieses Geräts zu erhöhen.",
-  "setEntropy.success": "Erzeugung der Entropie erfolgreich abgeschlossen",
+  "setEntropy.headline": "Erhöhen Sie die Sicherheit Ihres Benutzerkontos",
```

Comment:
@rohan-wire this will be overwritten by our translation process right after merge. The string that are not in `en-US` need to be changed in the crowdin interface in order to land in the webapp

Reply:
OK, @atomrc
Should I withdraw this PR and reassign the Jira ticket to someone else?

Reply:
You should only change the English version. Other languages will be edited directly in Crowdin.